mail/admin.py

```diff
@@ -0,0 +1,25 @@
+"""
+Admin views for Mail app
+"""
+
+from django.contrib import admin
+
+from mail.models import FinancialAidEmailAudit
+
+
+class FinancialAidEmailAuditAdmin(admin.ModelAdmin):
```

Comment:
Setting `has_change_permission` to false causes the model to not show up in the django admin.  As noted in #1258, there is an open django issue for this (https://code.djangoproject.com/ticket/8936), so, consistent with the financialaidaudit, we just set the fields to be readonly